src/lib/Scenes/Onboarding/OnboardingSocialPick.tsx

```diff
@@ -55,106 +72,128 @@ export const OnboardingSocialPick: React.FC<OnboardingSocialPickProps> = ({ mode
   }
 
   return (
-    <Flex justifyContent="center" alignItems="center" flex={1} px={1.5} backgroundColor="white">
+    <Flex justifyContent="center" flex={1} backgroundColor="white">
       <BackButton
         onPress={() => {
           navigation.goBack()
         }}
       />
-      <Join separator={<Spacer height={60} />}>
-        <Text variant="lg">{mode === "login" ? "Log in" : "Create account"}</Text>
+      <Flex px={1.5}>
+        <Join separator={<Spacer height={60} />}>
+          <Text variant="xxl">{mode === "login" ? "Log in" : "Create account"}</Text>
```

Comment:
```suggestion
          <Text variant="xxl">{mode === "login" ? "Log In" : "Create Account"}</Text>
```
I suppose we are following capitalize={'words'} approach for everything.